src/services/album-art-api.js

```diff
@@ -0,0 +1,7 @@
+const getAlbumArt = id => {
+  return fetch(`http://coverartarchive.org/release/${id}/front`, {}).then(res =>
+    res.json()
+  );
```

Comment:
stylistically, perhaps putting the `.then` on a new line would look a bit cleaner?